src/IR.cpp

```diff
@@ -583,6 +583,8 @@ Call::ConstString Call::if_then_else = "if_then_else";
 Call::ConstString Call::glsl_texture_load = "glsl_texture_load";
 Call::ConstString Call::glsl_texture_store = "glsl_texture_store";
 Call::ConstString Call::glsl_varying = "glsl_varying";
+Call::ConstString Call::coordinates_load = "coordinates_load";
```

Comment:
I like image_load/store more than coordinate_load/store, as "coordinate" can be read as a verb here.